### PR TITLE
Media lib with no mui error

### PR DESCRIPTION
**Pull request by frimpongopoku**
This PR contains the following: 

1. "Add To Gallery Page " 
The page allows an admin to upload images freely to be used later. 

2. "All images Page" 
Here community admins can see all images from their communities, and filter through them with some initial filters I have provided. On click, images show which events, actions, testimonials use them. 

Currently, the media library has been added to the form generator and is working properly.  Now, developers can specify "media-library" as fieldType and the form generator can be used. Everything is working nicely

I have not replaced in old form generator upload functionality with the new media library. Doing this would require corresponding backend changes to the way everything works. That can be a different task & PR.

This is only manually tested. I will write tests in the following PRs with other feedback I am given from here....

### CODE REVIEW

File: app/containers/pageListAsync.js [hunk 1]
```diff
@@ -1,41 +1,83 @@
...
-export const Impact = loadable(() => import('./MassEnergizeSuperAdmin/Pages/Impact'), {
-  fallback: <Loading />,
-});
-export const ImpactPage = loadable(() => import('./MassEnergizeSuperAdmin/Pages/ImpactPage'), {
-  fallback: <Loading />,
-});
-export const SuperAllActions = loadable(() => import('./MassEnergizeSuperAdmin/Pages/AllActions'), {
-  fallback: <Loading />,
-});
+export const AddToGallery = loadable(
```
Comment (BradHN1) [inline app/containers/pageListAsync.js:5]: @frimpongopoku - I don't like it when your auto-formatter does stuff like this.  It makes it difficult to review changes, and often makes the code less readable in my opinion.  Can you please turn it off?

File: package.json [hunk 2]
```diff
@@ -93,8 +92,8 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.12.1",
     "@babel/polyfill": "7.4.3",
     "@date-io/moment": "^1.3.13",
-    "@material-ui/core": "^3.9.4",
-    "@material-ui/icons": "^4.9.1",
+    "@material-ui/core": "^3.0.0",
```
Comment (BradHN1) [inline package.json:95]: Please document whether the material-ui core needs to revert from 3.9.4 (2 years old) to 3.0.0 (3 years old, a version no-one uses), and whether material-ui icons needs to revert from 4.9.1 (2 years old) to 3.0.2 (3 years old).  I don't want to revert to older versions unless absolutely necessary

**Review by BradHN1 — COMMENTED** [review]
This code tests fine and I want to merge to dev
Before merging it I need to know about whether material-ui core and icons really needs to revert to older versions

**Review by BradHN1 — APPROVED** [review]
Good to go.  Fixed a couple build errors.

File: app/containers/pageListAsync.js (hunk 1 above)
Comment (frimpongopoku) [reply app/containers/pageListAsync.js:5]: Hi @BradHN1, I will try my best to check and "unformat" things before I make PRs. Just like you, I am also unable to read and understand the code properly when  it isnt arranged. So I will just try to "unformat" when I am done to help you as well. 
Nevertheless, I do not think we should be fighting against formatters 🤣 🤣 . We cant always be aligning brackets and tracking indentations ourselves when we can just offset it to a formatter; plus, a formatter does it faster, and does it the way its meant to be done. It just takes some getting used to 🤗 🤣 .

**Pull request merged**